src/org.xtuml.bp.docgen/src/org/xtuml/bp/docgen/generator/Generator.java

```diff
@@ -424,8 +422,18 @@ private static void runXsltproc(IProject project, String workDir)
 
         IFileSystem fileSystem = EFS.getLocalFileSystem();
         
-        IFileStore srcxsl = fileSystem.getStore(new File(homedir + DOCGEN_DIR + xslfile).toURI());
-        IFileStore tgtxsl = fileSystem.getStore(new File(workingDir + xslfile).toURI());
+        // move docgen into the same folder as docbook.xsl.  This is done because docgen.xsl includes docbook.xsl and
+        // for that include to be resolved we have to have one of 3 things:
+        // 1) docgen.xsl and docbooks.csl are in the same folder (this is what we have done)
+        // 2) Use the xsltproc --path option to add a path for docbook.xsl
+        //    Note: This was preferred, but failed because at runtime the --path option was seen as an invalid
+        //          option.  this was odd because it works on the command-line, but it does not work, at least in linux,
+        //          at runtime.
+        // 3) modify the include specific in doc.xml to have a full of relative path to docbook.xsl.
+        //    Note: This was the option used prior to v5.0 of the tool.  Version 5.0 removed the post-processing of installation
+        //          that performed this modifiation, so this option is no longer possible, and is was not a desirable option anyway.
+        IFileStore srcxsl = fileSystem.getStore(new File(homedir + DOCGEN_DIR + DOCGEN_XSL).toURI());
+        IFileStore tgtxsl = fileSystem.getStore(new File(docbook_folder + "/" + DOCGEN_XSL).toURI());

```

Comment:
The previous implementation had to copy this file in every time "Create Documentation" ran because it was copying the xsl file from the installed location into the project being docgen'd.  So, that had to happen dynamically every time you ran docgen.  But, now you are just copying from the installed location to another installed location.  So, why not just have the build packaging put the file into the "right" place rather than this other place where you have to move it at runtime?  Then it's done and you can remove this runtime manipulation code completely.